app/models/user.rb

```diff
@@ -172,13 +174,22 @@ def mfa_enabled?
     !mfa_disabled?
   end
 
+  def webauthn_enabled?
+    credentials.any?
+  end
+
   def disable_mfa!
     mfa_disabled!
     self.mfa_seed = ""
     self.mfa_recovery_codes = []
     save!(validate: false)
   end
 
+  def disable_webauthn!
+    credentials.first.destroy!
```

Comment:
Consider if this should `credentials.destroy_all`, or if the credentials association ought to be a `has_one` instead of `has_many`.

Reply:
In the end there should be multiple credentials so `has_many`.

And about destruction, it will probably make sense to have a delete for each of the credentials. Not sure we'll keep this `disable_webauthn!` method to delete all at once.